src/ducks/Studio/AdvancedView/Histogram/hooks.js

```diff
@@ -1,14 +1,14 @@
 import { useState, useEffect } from 'react'
 import { linearScale } from '@santiment-network/chart/scales'
 import { useQuery } from '@apollo/react-hooks'
-import { HISTOGRAM_DATA_QUERY } from './gql'
+import { HISTOGRAM_DATA_QUERY, PROJECT_PRICE_QUERY } from './gql'
 
 const Chart = {
-  height: 342,
+  height: 50,
   top: 0
 }
 
-function formatHistogramData (data) {
+function formatHistogramData (data, price) {
   const { length } = data
 
   let max = 0
```

Comment:
I am thinking the size of data is predictable and small no more than 20.
Maybe better to use one liner to make less code and inc readability.
Math.max(...data.map(el=>el.value))
? @DmitriVanGuard

Reply:
I'm mean, this is a 3 line algorithm (loop, value check, assignment), which intention is pointed out beforehand with a variable called `max`.

The difference in speed for the array with 10 items: ![image](https://user-images.githubusercontent.com/25135650/77818773-57707c00-70e6-11ea-8254-54ad1990e723.png)
For a simple task, it's a huge difference.